ERROR - 2024-05-10 14:39:29 --> mysqli_sql_exception: ocupado,C19 in C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO asi...', 0)
#1 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO asi...')
#2 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO asi...')
#3 C:\xampp\htdocs\Bib\app\Models\Principal.php(110): CodeIgniter\Database\BaseConnection->query('INSERT INTO asi...', Array)
#4 C:\xampp\htdocs\Bib\app\Controllers\ControllerPrincipal.php(189): App\Models\Principal->ValidarAll(Array)
#5 C:\xampp\htdocs\Bib\system\CodeIgniter.php(942): App\Controllers\ControllerPrincipal->ValidarAll('0323150006', 'C19')
#6 C:\xampp\htdocs\Bib\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControllerPrincipal))
#7 C:\xampp\htdocs\Bib\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Bib\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2024-05-10 14:39:45 --> mysqli_sql_exception: ocupado,E12 in C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO asi...', 0)
#1 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO asi...')
#2 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO asi...')
#3 C:\xampp\htdocs\Bib\app\Models\Principal.php(110): CodeIgniter\Database\BaseConnection->query('INSERT INTO asi...', Array)
#4 C:\xampp\htdocs\Bib\app\Controllers\ControllerPrincipal.php(189): App\Models\Principal->ValidarAll(Array)
#5 C:\xampp\htdocs\Bib\system\CodeIgniter.php(942): App\Controllers\ControllerPrincipal->ValidarAll('0323150010', 'E12')
#6 C:\xampp\htdocs\Bib\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControllerPrincipal))
#7 C:\xampp\htdocs\Bib\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Bib\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2024-05-10 19:10:30 --> mysqli_sql_exception: ocupado,E15 in C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\Bib\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO asi...', 0)
#1 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO asi...')
#2 C:\xampp\htdocs\Bib\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO asi...')
#3 C:\xampp\htdocs\Bib\app\Models\Principal.php(110): CodeIgniter\Database\BaseConnection->query('INSERT INTO asi...', Array)
#4 C:\xampp\htdocs\Bib\app\Controllers\ControllerPrincipal.php(189): App\Models\Principal->ValidarAll(Array)
#5 C:\xampp\htdocs\Bib\system\CodeIgniter.php(942): App\Controllers\ControllerPrincipal->ValidarAll('0322140018', 'D11')
#6 C:\xampp\htdocs\Bib\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControllerPrincipal))
#7 C:\xampp\htdocs\Bib\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Bib\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
